dbt/models/marts/students/dim_student_surveys.sql

```diff
@@ -0,0 +1,89 @@
+-- model: dim_student_surveys 
+
+with 
+contained_levels as (
+    select * 
+    from {{ ref('stg_dashboard__contained_levels') }}
+),
+
+levels as (
+    select 
+        *,          
+        case 
+            when level_type = 'multi' then json_array_length(json_extract_path_text(properties, 'answers'))
+            when level_type = 'freeresponse' 	then 1 
+            when level_type = 'external'		then 0 
+        end 	as num_response_options
+
+    from {{ ref('stg_dashboard__levels') }}
+),
+
+script_levels as (
+    select *
+    from {{ ref('stg_dashboard__script_levels') }}
+),
+
+levels_script_levels as (
+    select * 
+    from {{ ref('stg_dashboard__levels_script_levels') }}
+),
+
+scripts as (
+    select * 
+    from {{ ref('stg_dashboard__scripts') }}   
+),
+
+course_structure as (
+    select * 
+    from {{ ref('dim_course_structure') }}
+    where course_name in ('csa','csd','csp')
+),
+
+combined as (
+    select 
+        lower(gl.level_name)            as survey_name,
```

Comment:
```suggestion
        cs.content_area,
        cs.course_name,
        cs.script_id,
        cs.script_name,
        cs.unit, 
        cs.topic_tags,
        cs.version_year,
        col.level_group_level_id as survey_level_id,
        lower(gl.level_name)            as survey_name,
```
Reordering for ease of use